google/cloud/spanner/integration_tests/client_integration_test.cc

```diff
@@ -70,6 +70,31 @@ class ClientIntegrationTest : public spanner_testing::DatabaseIntegrationTest {
 
 std::unique_ptr<Client> ClientIntegrationTest::client_;
 
+class PgClientIntegrationTest
+    : public spanner_testing::PgDatabaseIntegrationTest {
+ protected:
+  static void SetUpTestSuite() {
+    spanner_testing::PgDatabaseIntegrationTest::SetUpTestSuite();
+    client_ = absl::make_unique<Client>(MakeConnection(GetDatabase()));
+  }
+
+  void SetUp() override {
+    if (UsingEmulator()) return;
+    auto commit_result = client_->Commit(
+        Mutations{MakeDeleteMutation("Singers", KeySet::All())});
+    ASSERT_STATUS_OK(commit_result);
+  }
+
+  static void TearDownTestSuite() {
+    client_ = nullptr;
+    spanner_testing::PgDatabaseIntegrationTest::TearDownTestSuite();
+  }
+
+  static std::unique_ptr<Client> client_;
```

Comment:
`<mumble>` something something non-trivial destructors something something Google Style Guide something `</mumble>`.

But we already have one of those here 🤷